docs/README.md

```diff
@@ -1,28 +1,152 @@
+installs a Stacks with permissions to watch resources at a cluster level, across
+all namespaces.
 
-The full architecture and vision of the Crossplane project is described in depth in the [architecture document](https://docs.google.com/document/d/1whncqdUeU2cATGEJhHvzXWC9xdK29Er45NJeoemxebo/edit?usp=sharing). It is the best place to learn more about how Crossplane fits into the Kubernetes ecosystem, the intended use cases, and comparisons to existing projects.
+Infra Stacks are covered in both the [Using Services](using-services.md) and
+[Using Stacks](using-stacks.md) guides. 
+
+### Application Stacks 
+App Stacks depend on Infra Stacks like
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws), or
+[stack-azure](https://github.com/crossplaneio/stack-azure) to provide managed
+services via the Kubernetes API.
+
+App Stacks are pre-built and published to the [Stacks
+registry](https://hub.docker.com/search?q=crossplane&type=image) where they can
+be installed by an app team using a
+[`StackInstall`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow)
+Kubernetes API kind or with the [`stack
+install`](https://github.com/crossplaneio/crossplane-cli#usage) command that
+installs Stacks with permissions only within a single namespace.
+
+The [Using Stacks](using-stacks.md) guide covers how to deploy a [portable
+Wordpress App Stack](https://hub.docker.com/r/crossplane/sample-stack-wordpress)
+into multiple clouds using Infra Stacks for
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws),
+[stack-azure](https://github.com/crossplaneio/stack-azure). 
+
+### Build your own Stack 
+The [Developer Guide](developer-guide.md) shows how to
+build and extend Infra Stacks and App Stacks. 
+
+## Crossplane Workloads 
+Crossplane includes a workload scheduler that can factor
+a number of criteria including capabilities, availability, reliability, cost,
+regions, and performance while deploying workloads and their resources. The
+scheduler works alongside specialized resource controllers to ensure policies
+set by administrators are honored. Complex workloads can be modeled as a
+[`KubernetesApplication`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-complex-workloads.md)
+resource as done by the [portable Wordpress App Stack
+controller](https://github.com/crossplaneio/sample-stack-wordpress) which
+automates the lifecycle of a
+[`KubernetesApplication`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-complex-workloads.md)
+including provisioning and secure connectivity to a managed `MySQLInstance`.
+
+## Crossplane Clusters 
+Crossplane supports dynamic provisioning of managed
+Kubernetes clusters from a single control plane with consistent multi-cluster
+best-practice configuration and secure connectivity between target Kubernetes
```

Comment:
Can you clarify what Crossplane does to ensure "consistent multi-cluster best-practice configuration"?

Reply:
using the same cluster resource class and secure networking resource for repeatable configurations across environments, e.g. dev, staging in different namespaces